src/main/java/haveric/recipeManager/Vanilla.java

```diff
@@ -142,7 +142,7 @@ protected static void init() {
             initialRecipes.put(new FuelRecipe(Material.FISHING_ROD, 15), info);
             initialRecipes.put(new FuelRecipe(Material.SIGN, 10), info);
             initialRecipes.put(new FuelRecipe(Material.BOWL, 5), info);
-            initialRecipes.put(new FuelRecipe(Material.WOODEN_DOOR, 10), info);
+            initialRecipes.put(new FuelRecipe(Material.WOOD_DOOR, 10), info); // was WOODEN_DOOR, which is block type, not item type...

```

Comment:
AFAIK it was always the case, but no, I do not know. I can do some sleuthing later.